test/hotspot/jtreg/compiler/vectorization/runner/ArrayShiftOpTest.java

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2022, 2023, Arm Limited. All rights reserved.
- * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2024, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Leave that copyright untouched but add a Rivos line instead.

Reply:
Thanks! It's updated.